Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2421,7 +2421,7 @@ extension BrowserViewController: ClientPickerViewControllerDelegate, Instruction
         self.popToBVC()
     }
 
-    func clientPickerViewController(_ clientPickerViewController: ClientPickerViewController, didPickClients clients: [RemoteClient]) {
+    func clientPickerViewController(_ clientPickerViewController: ClientPickerViewController, didPickDevices devices: [RemoteDevice]) {
```

Comment:
follow-up: we should rename `ClientPickerViewController` to `DevicePickerViewController`